### PR TITLE
Fix dependency

### PR DESCRIPTION
**Pull request by michalzagrodzki**
1. Add missing dependency - axios and vue-axios
2. Correct order of imported elements in main.js - according to eslint.
3. Replace 'require' with 'import' in tagsService.js and postsService.js

### CODE REVIEW

**Review by mchandleraz — APPROVED** [review]
Thanks Michal!

**Comment by michalzagrodzki** [comment]
Glad you merged it. Waiting for approval of next PR.

**Pull request merged**
